agent/installer/registry.go

```diff
@@ -11,18 +11,26 @@ import (
 type osk8sInstaller interface{}
 type k8sInstallerMap map[string]osk8sInstaller
 type osk8sInstallerMap map[string]k8sInstallerMap
-type filterBundlePair struct {
+type filterOsBundlePair struct {
 	osFilter string
 	osBundle string
 }
-type filterBundleList []filterBundlePair
+
+type filterK8sBundle struct {
+	k8sFilter string
+}
+
+type filterBundleList []filterOsBundlePair
```

Comment:
```suggestion
type filterOSBundleList []filterOsBundlePair
```

Feel free to include this in one of the future PRs

Reply:
Will do it right away, thanks!